docs/installation.rst

```diff
@@ -10,6 +10,13 @@ Python environment, use the pip command:
 
    pip install pysd
 
+Installing with conda
+---------------------
+To install PySD with conda into a conda environment, use the following command:
```

Comment:
It may be worth mentioning that this doesn't use the standard conda channel, which is why we have the `-c conda-forge`. It's not important, but may save a few questions down the line.

Reply:
Yes :)